simulator-starter/src/main/java/org/citrusframework/simulator/scenario/ScenarioDesigner.java

```diff
@@ -16,22 +16,32 @@
 
 package org.citrusframework.simulator.scenario;
 
+import org.citrusframework.DefaultTestCaseRunner;
+import org.citrusframework.TestActionBuilder;
+import org.citrusframework.TestActionContainerBuilder;
+import org.citrusframework.actions.ReceiveMessageAction;
+import org.citrusframework.actions.SendMessageAction;
+import org.citrusframework.container.FinallySequence;
+import org.citrusframework.container.TestActionContainer;
+import org.citrusframework.context.TestContext;
+import org.citrusframework.exceptions.CitrusRuntimeException;
+import org.citrusframework.http.actions.HttpServerRequestActionBuilder;
+import org.citrusframework.http.actions.HttpServerResponseActionBuilder;
 import org.citrusframework.simulator.correlation.CorrelationHandlerBuilder;
 import org.citrusframework.simulator.correlation.CorrelationManager;
 import org.citrusframework.simulator.http.HttpScenarioActionBuilder;
 import org.citrusframework.simulator.ws.SoapScenarioActionBuilder;
+import org.citrusframework.spi.ReferenceResolver;
 import org.springframework.context.ApplicationContext;
 
-import com.consol.citrus.context.TestContext;
-import com.consol.citrus.dsl.builder.ReceiveMessageActionBuilder;
-import com.consol.citrus.dsl.builder.SendMessageActionBuilder;
-import com.consol.citrus.dsl.design.DefaultTestDesigner;
-import com.consol.citrus.spi.ReferenceResolver;
+import java.util.Stack;
+
+import static org.citrusframework.container.FinallySequence.Builder.doFinally;
 
 /**
  * @author Christoph Deppisch
  */
-public class ScenarioDesigner extends DefaultTestDesigner {
+public class ScenarioDesigner extends DefaultTestCaseRunner {
```

Comment:
the whole concept around "Designers" has been removed from the core, I think. this class would be removed and the `ScenarioRunner` would be the only remaining option, right? it does not differ much from the designer, tho.